---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

{% if author.googlescholar %}
  You can also find my articles on <u><a href="{{author.googlescholar}}">my Google Scholar profile</a>.</u>
{% endif %}

<!-- {% include base_path %}

{% for post in site.publications reversed %}
  {% include archive-single.html %}
{% endfor %} -->

**Journal Papers**

* J. Ding<sup>\*</sup>, H. Wen<sup>\*</sup>, **W. Tang<sup>\*</sup>**, R. Liu<sup>\*</sup>, et al. \
  **DANCE: A Deep Learning Library and Benchmark Platform for Single-cell Analysis.** [[link]](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03211-z) \
  *Genome Biology*, 2024.

* D. Molho<sup>\*</sup>, J. Ding<sup>\*</sup>, **W. Tang<sup>\*</sup>**, et al. \
  **Deep Learning in Single-Cell Analysis.** [[pdf]](https://dl.acm.org/doi/abs/10.1145/3641284) \
  *ACM Transactions on Intelligent Systems and Technology (ACM TIST)*, 2024.

* S. Wang, J. Huang, Z. Chen, Y. Song, **W. Tang**, et al. \
  **Graph Machine Learning in the Era of Large Language Models (LLMs).** [[link]](https://dl.acm.org/doi/10.1145/3732786) \
  *ACM Transactions on Intelligent Systems and Technology (ACM TIST)*, 2025.

* J. Ding, J. Venegas, Q. Lu, Y. Wang, W. Jin, H. Wen, R. Liu, **W. Tang**, et al. \
  **SpatialCTD: A Large-Scale Tumor Microenvironment Spatial Transcriptomic Dataset to Evaluate Cell Type Deconvolution for Immuno-Oncology.** [[pdf]](https://www.biorxiv.org/content/10.1101/2023.04.11.536333v1) \
  *Journal of Computational Biology*, 2024.


**Conference Papers**

* **W. Tang**, et al. \
  **Cross-Domain Graph Data Scaling: A Showcase with Diffusion Models.** [[pdf]](https://arxiv.org/abs/2406.01899) \
  *NeurIPS*, 2025.

* H. Wen<sup>\*</sup>, **W. Tang<sup>\*</sup>**, et al. \
  **CellPLM: Pre-training of Cell Language Model Beyond Single Cells.** [[pdf]](https://openreview.net/forum?id=BKXvPDekud) \
  *ICLR*, 2024.

* H. Mao<sup>\*</sup>, Z. Chen<sup>\*</sup>, **W. Tang**, et al. \
  **Position: Graph Foundation Models Are Already Here.** [[pdf]](https://arxiv.org/abs/2402.02216) \
  *ICML* (**Spotlight**), 2024.

* **W. Tang<sup>\*</sup>**, H. Wen<sup>\*</sup>, R. Liu<sup>\*</sup>, et al. \
  **Single-Cell Multimodal Prediction via Transformers.** [[pdf]](https://arxiv.org/abs/2303.00233) \
  *CIKM*, 2023.

* H. Li, W. Jin, G. Skenderi, H. Shomer, **W. Tang**, et al. \
  **Sub-graph Based Diffusion Model for Link Prediction.** \
  *Learning on Graphs Conference (LoG)*, 2024.


**Preprints**

* **W. Tang<sup>\*</sup>**, R. Liu<sup>\*</sup>, et al. \
  **A General Single-Cell Analysis Framework via Conditional Diffusion Generative Models.** [[pdf]](https://www.biorxiv.org/content/10.1101/2023.10.13.562243v1) \
  *bioRxiv*, 2023.

* H. Wen, **W. Tang**, et al. \
  **Single Cells Are Spatial Tokens: Transformers for Spatial Transcriptomic Data Denoising.** [[pdf]](https://arxiv.org/abs/2302.03038) \
  *arXiv*, 2023.

* J. Ding, Z. Xing, Y. Wang, R. Liu, S. Liu, Z. Huang, **W. Tang**, et al. \
  **DANCE 2.0: Transforming Single-Cell Analysis from Black Box to Transparent Workflow.** \
  *bioRxiv*, 2025.

* Y. Wang<sup>\*</sup>, J. Ding<sup>\*</sup>, L. Wu, A. Wardhani, P. Danaher, Q. Lu, H. Wen, **W. Tang**, et al. \
  **MEM-GAN: A Pseudo Membrane Generator for Single-cell Imaging in Fluorescent Microscopy.** \
  *bioRxiv*, 2023.
